lib/puppet/type/rds_db_parameter_group.rb

```diff
@@ -0,0 +1,25 @@
+Puppet::Type.newtype(:rds_db_parameter_group) do
+  @doc = 'Type representing an RDS DB Paremeter group'
+
+  ensurable
+
+  newparam(:name, namevar: true) do
+    desc 'the name of the DB Parameter Group (also known as the db_parameter_group_name)'
+  end
+
+  newproperty(:description) do
+    desc 'the description of a DB parameter group'
+  end
+
+  newproperty(:db_parameter_group_family) do
```

Comment:
Would this make sense just as family? Given it's already in the context of a db_parameter_group